src/confdb/confdb.c

```diff
@@ -1310,9 +1310,15 @@ static int confdb_get_domain_internal(struct confdb_ctx *cdb,
         }
     } else {
         /* default */
-        domain->case_sensitive = true;
-        domain->case_preserve = true;
+        if (strcasecmp(domain->provider, "ad") == 0) {
```

Comment:
Why is this patch needed, or what is the test case here? It would be nice to at least add a comment to the code or the commit message..

Either way, one of the `case_sensitive` below is probably meant to say `case_preserving`.